lib/parser.js

```diff
@@ -79,14 +84,14 @@ Parser.prototype._preprocess = function (element, context, config) {
     let isIncludeSrcMd = utils.getExtName(filePath) === 'md';
 
     let actualFilePath = filePath;
-    if (!utils.fileExists(filePath)) {
+    if (utils.fileExists(filePath)) {
+      this.staticIncludeSrc.push({from: context.cwf, to: actualFilePath});
+    } else {
       let boilerplateFilePath = calculateBoilerplateFilePath(filePath, config);
       actualFilePath = utils.fileExists(boilerplateFilePath) ? boilerplateFilePath : actualFilePath;
+      this.boilerplateIncludeSrc.push({from: context.cwf, to: actualFilePath});
```

Comment:
This will add the original file path if the boilerplate file does not exist.
Should it be the following?
```js
if (utils.fileExists(boilerplateFilePath)) {
  actualFilePath = boilerplateFilePath;
  this.boilerplateIncludeSrc.push({from: context.cwf, to: boilerplateFilePath});
 }
```

Reply:
Thanks for pointing this out!